modules/grandpa/src/lib.rs

```diff
@@ -693,7 +693,7 @@ mod tests {
 				CurrentAuthoritySet::<TestRuntime>::get().authorities,
 				init_data.authority_list
 			);
-			assert_eq!(IsHalted::<TestRuntime>::get(), false);
+			assert!(!IsHalted::<TestRuntime>::get());
```

Comment:
Tbh I'm not a fan of this Clippy suggestion, the other was easier to read (no double !!), but 🤷‍♂️

Reply:
It gives much better debugging info if it fails (shows the entire expression instead of `expected true, got false`).